easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-5.1.2-2023b.yml

```
@@ -29,3 +29,19 @@ easyconfigs:
         # see https://github.com/easybuilders/easybuild-easyconfigs/pull/23155
         from-commit: b35081d4454f54996108088780f6554739dc4891
   - Ruby-3.4.2-GCCcore-13.2.0.eb
+  - Boost.Python-NumPy-1.83.0-gfbf-2023b.eb:
+      options:
+        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/23155
+        from-commit: b35081d4454f54996108088780f6554739dc4891
+  - LoSoTo-2.5.0-foss-2023b.eb:
+      options:
+        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/23155
+        from-commit: b35081d4454f54996108088780f6554739dc4891
+  - PyTables-3.9.2-foss-2023b.eb:
+      options:
+        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/23155
+        from-commit: b35081d4454f54996108088780f6554739dc4891
```

This one is even included in EB, so let's get rid of the `from-commit`?
```suggestion
  - PyTables-3.9.2-foss-2023b.eb
```